### DISCUSSION TITLE
standardize on amd64 for x86_64 artifact names

### DISCUSSION THREAD

**Discussion by shmick** [DNSCrypt/dnscrypt-proxy#2072] [Ideas]
The 64bit linux artifact has the suffix `x86_64` while the rest of the artifacts use `amd64`

```
dnscrypt-proxy-linux_x86_64-2.1.1.tar.gz
dnscrypt-proxy-netbsd_amd64-2.1.1.tar.gz
```

When using utils like docker buildx, `amd64` is the architecture type for `x86_64` and `amd64`, and exposes that as the `TARGETARCH` environment variable.

If we try to download the build artifact using something like:

```URL="https://github.com/jedisct1/dnscrypt-proxy/releases/download/${VERSION}/dnscrypt-proxy-linux_${TARGETARCH}-${VERSION}.tar.gz"```

This will fail since TARGETARCH = amd64 and `dnscrypt-proxy-linux_amd64-2.1.1.tar.gz` does not exist.

**Comment by jedisct1** [comment]
This is unfortunate that the architecture is sometimes called `x86_64` and sometimes called `amd64`.

Linux calls it `x86_64`. But on BSD, the architecture is called `amd64`:

```sh
$ arch
OpenBSD.amd64
```

The package name matches what people are used to for their operating system.
